ingress/controllers/gce/controller/utils.go

```
@@ -315,3 +320,40 @@ func (t *GCETranslator) toNodePorts(ings *extensions.IngressList) []int64 {
 	}
 	return knownPorts
 }
+
+func getZone(n api.Node) string {
```

Why return a empty zone instead of err? 
It may hide underlying error